### PR TITLE
task(VENLY-3610) - add classes for IMX sign

### PR DESCRIPTION
**Pull request by ZhamakochyanErik**

### CODE REVIEW

File: provider-api-model/src/main/java/network/arkane/provider/sign/ImxEip712Signable.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import network.arkane.provider.sign.domain.Signable;
+
+@Data
+@NoArgsConstructor
+public class ImxEip712Signable implements Signable {
+    private String data;
+
+    @Builder(builderClassName = "ImxEip712SignableBuilder")
```
Comment (maciejwitwicki) [inline provider-api-model/src/main/java/network/arkane/provider/sign/ImxEip712Signable.java:13]: I think the effect would be the same if you use @Builder annotation at the top of the class?

File: provider-api-model/src/main/java/network/arkane/provider/sign/ImxRawSignable.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import network.arkane.provider.sign.domain.Signable;
+
+@Data
+@NoArgsConstructor
+public class ImxRawSignable implements Signable {
+    private String data;
+
+    @Builder(builderClassName = "ImxRawSignableBuilder")
```
Comment (maciejwitwicki) [inline provider-api-model/src/main/java/network/arkane/provider/sign/ImxRawSignable.java:13]: I think the effect would be the same if you use @builder annotation at the top of the class?

File: provider-api-model/src/main/java/network/arkane/provider/sign/ImxRawVerifiable.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import network.arkane.provider.sign.domain.Verifiable;
+
+@Data
+@NoArgsConstructor
+public class ImxRawVerifiable implements Verifiable {
+    private String signature;
+    private String message;
+    private String address;
+
+    @Builder(builderClassName = "ImxRawVerifiableBuilder")
```
Comment (maciejwitwicki) [inline provider-api-model/src/main/java/network/arkane/provider/sign/ImxRawVerifiable.java:15]: I think the effect would be the same if you use @builder annotation at the top of the class?

**Review by maciejwitwicki — CHANGES_REQUESTED** [review]

File: provider-api-model/src/main/java/network/arkane/provider/sign/ImxEip712Signable.java (hunk 1 above)
Comment (ZhamakochyanErik) [reply provider-api-model/src/main/java/network/arkane/provider/sign/ImxEip712Signable.java:13]: yes I agree with you effect will  be the same
I just added on the top of constructor for keeping  consistency  
other Signable classes are written so and I wrote classes like them for keeping consistency

**Review by maciejwitwicki — APPROVED** [review]

**Review by aleksanderndoci — APPROVED** [review]

**Pull request merged**